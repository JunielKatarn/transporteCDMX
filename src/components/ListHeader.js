import React, { memo } from 'react'
import { View } from 'react-native'
import { sharedStyles } from '../styles'
import { Caption } from 'react-native-paper'

const ListHeader = ({ text }) => {
  return (
    <View style={[sharedStyles.mb3, sharedStyles.centerChild]}>
      <Caption>{text}</Caption>
    </View>
  )
}

export default memo(ListHeader)
