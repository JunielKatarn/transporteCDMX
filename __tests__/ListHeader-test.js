import React from 'react'
import { shallow } from 'enzyme'
import ListHeader from '../src/components/ListHeader'

describe('ListHeader', () => {
  it('it renders correctly', () => {
    const wrapper = shallow(<ListHeader text="Header text" />)
    expect(wrapper).toMatchSnapshot()
  })
})
