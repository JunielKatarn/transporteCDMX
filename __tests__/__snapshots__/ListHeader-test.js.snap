// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListHeader it renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "marginBottom": 17,
      },
      Object {
        "alignItems": "center",
        "justifyContent": "center",
      },
    ]
  }
>
  <Caption>
    Header text
  </Caption>
</View>
`;
